content/Memory_Map.md

```diff
@@ -27,7 +27,7 @@ The following addresses are supposed to be used as jump vectors:
 
 However, the memory may be used for any other purpose in case that your
 program doesn't use any (or only some) RST commands or interrupts. RST
-commands are 1-byte opcodes that work similar to CALL opcodes, except
+commands are 1-byte instructions that work similarly to CALL instructions, except
```

Comment:
I'd rather say that "`rst` is a 1-byte instruction that works similarly to the 3-byte `call` instruction, except that the destination address is restricted.". What do you think?

Reply:
I agree. Fixed.